docs/UserGuide.md

```diff
@@ -289,6 +293,17 @@ You will have to edit them using the [Editing Student Information](#642-editing-
 #### 6.1.4 Notes about truncation
    * If the user input is too long, it is automatically truncated when displayed.
    <br> e.g. if the user enters a `NAME` parameter that is 150 characters long, Jarvis will automatically truncate it to fit the given screen size.
+   
+#### 6.1.5 Notes about Summary feature
+   * As mentioned above under the section explaining the top bar, the `Summary` feature is updated when Jarvis is
+   launched and after a user command is successfully executed. As such, the values in the `Summary
+   ` will remain the same, even if
+   the current time has passed the date and time of a `Consultation` or `Mastery Check`. Also, if a `Mission` or
+   `Quest` is graded while Jarvis is open, the number of ungraded `Mission` and `Quests` will remain the same.
+   * To get the updated values, simply re-start Jarvis or type in a command and hit the enter key on your keyboard.
```

Comment:
I feel that this bullet point under this section 6.1.5 is quite important. Maybe we can give a markdown box for caution?

Reply:
That is true, great suggestion! @peironggg could you help with this since you are working on the UG currently? thank you!